src/pipx/emojis.py

```diff
@@ -1,15 +1,7 @@
 import os
 import sys
 
-
-def strtobool(val: str) -> bool:
```

Comment:
Same for this change. Could you bundle them in a separate PR?

Reply:
It felt wrong using `strtobool` from `emojis.py` in `paths.py` for something completely emoji unrelated, so I moved it to utils. This required the import changes to fix a circular import.
Do you think I should stick to importing it from emojis for this PR?

Reply:
Probably, yes; this and the circular import problem mentioned in #1333 could be resolved in one go.

Reply:
Removed it from this PR